apps/main/src/loan/components/PageLoanCreate/Page.tsx

```diff
@@ -125,9 +126,6 @@ const Page = () => {
         fetchInitial(curve, isLeverage, market)
         void fetchLoanDetails(curve, market)
         setLoaded(true)
```

Comment:
Think you can reduce a nesting here with `if (isHydrated && curve && market) {` above?